### DISCUSSION TITLE
My config eww

### DISCUSSION THREAD

**Discussion by Paladin1991** [elkowar/eww#174] [Show and tell]
This is my xmonad, with some things done with eww. Really incredible.
![2021-05-16-164249_1600x900_scrot](https://user-images.githubusercontent.com/77540915/118410505-234b7800-b666-11eb-91d2-8f4311f5cd7d.png)
![2021-05-16-164303_1600x900_scrot](https://user-images.githubusercontent.com/77540915/118410523-38c0a200-b666-11eb-8183-044b2a47add3.png)
![2021-05-16-164339_1600x900_scrot](https://user-images.githubusercontent.com/77540915/118410525-3c542900-b666-11eb-901d-4815c0d78075.png)
![2021-05-16-164452_1600x900_scrot](https://user-images.githubusercontent.com/77540915/118410530-3fe7b000-b666-11eb-9f5b-9fdc292cd9aa.png)
![2021-05-16-164514_1600x900_scrot](https://user-images.githubusercontent.com/77540915/118410532-42e2a080-b666-11eb-9ed7-60b8339a2562.png)

**Comment by hmm-yes** [comment]
this is cooool

**Comment by Babkock** [comment]
Wow. Looks really great

**Comment by VuiMuich** [comment]
@Paladin1991 is this from your Paladin1991/Last-Xmonad repo?

**Reply by Paladin1991** [reply]
Yeah man, stop with this new eww configuration file, you'll have to port tufa to the new configuration.
